Added updates requested in issue 631. (#685)
astronomer/docs: astro/airflow-alerts.md

### Integrate with Amazon SES

Use your existing Amazon SES instance to send Airflow alerts by email.

1. Sign in to the AWS Management Console and open the Amazon SES console at https://console.aws.amazon.com/ses/.

2. Click the menu icon and then click **Verified Identities**.

> 3. Optional. Complete one of the following tasks:

Reviewer: ```suggestion
3. Complete one of the following tasks to set up an email address:
```
One of these steps must be completed in order to use SES, so I don't think "Optional" is accurate here.

Author: @jwitz according to @ben-astro, this step is optional. Users might already have existing email addresses they want to use.

    - To confirm the email account is working and the Amazon SES service can send emails to it, click an email address in the **Identity** column and then click **Send test email**. When the owner of the email address receives the test email, they need to select the validation link in the email. 
    - To add a new email address, click **Create Identity**, add the email address, and then click **Create Identity**.
    
    For email alerts, Astronomer recommends using one email address as the sender and a second email address as the recipient. All email addresses must be verified. 
    

>     See  [Creating an email address identity](https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html#:~:text=of%20those%20Regions.-,Creating%20an%20email%20address%20identity,-Complete%20the%20following) and  [Verifying an email address identity](https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html#:~:text=address%20identity.-,Verifying%20an%20email%20address%20identity,-After%20you%E2%80%99ve%20created).

Reviewer: ```suggestion
    For more information about configuring Amazon SES, read [Creating an email address identity](https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html#:~:text=of%20those%20Regions.-,Creating%20an%20email%20address%20identity,-Complete%20the%20following) and  [Verifying an email address identity](https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html#:~:text=address%20identity.-,Verifying%20an%20email%20address%20identity,-After%20you%E2%80%99ve%20created) in the Amazon documentation.
```

Author: @jwitz I think this is something we need to discuss. Many of the links include far too much text. Also, I don't like "read" and prefer "see". I'll accept this for now, but we need to think about reducing and simplifying link text and avoiding "read".

In this instance, _in the Amazon documentation_ is unnecessary. If a user selects the link they'll notice they have moved to the Amazon documentation. Also, there is text preceding this link that provides the necessary context.

Reviewer: @kwcanuck Because we frequently link out to external docs, I think it's important to let a user know when they're going to be reaching an external site. This is mostly important in relation to Apache Airflow documentation, because I can definitely see a user being frustrated if they think they're going to a doc for some expert Astronomer advice and they end up on the same Apache Airflow doc they've used in the past. That said, I don't think it's a problem to extend this practice to other external documentation. 

I have no preference over "see" vs. "read" so I'll follow your lead there, but I do think we need some sort of clause that indicates "hey this is totally optional FYI information, you don't need to access these links unless you want to".

Author: @jwitz your explanation makes sense. One caveat to consider is that using the name of the documentation source (Amazon, for example) can cause issues in the unlikely event that the company changes its name or the referenced product name. Although the content will remain current, the outdated reference makes it appear stale.

4. Click **Account dashboard**.

5. In the **Simple Mail Transfer Protocol (SMTP) settings** pane, copy the value displayed below **SMTP endpoint**.

6. Obtain your Amazon SES SMTP credentials:
    - Click **Create SMTP credentials** in the **Simple Mail Transfer Protocol (SMTP) settings** pane.
    - Enter a name for your SMTP user in the **IAM User Name** field or accept the default value.
    - Click **Create** and then **Show User SMTP Security Credentials**.
    - Click **Download Credentials** or copy them and store them in a safe place.
    - Click **Close Window**.

> 7. Log in to the Astronomer Cloud UI, click **Deployments**, and then select an existing Airflow deployment.

Reviewer: ```suggestion
7. Log in to the Cloud UI, click **Deployments**, and then select an existing Deployment.
```

   
8. In the Environment Variables area, click **Edit Variables** and add these variables:

>     - ENV AIRFLOW__SMTP__SMTP_HOST: Enter the value you copied in step 5

Reviewer: ```suggestion
    - `ENV AIRFLOW__SMTP__SMTP_HOST`: Enter the value you copied in step 5.
```
We should use code formatting on the rest of these environment variables (also use punctuation because of the imperative statement). Good call on taking this out of a code fence though

Author: @jwitz @ben-astro expressed concern that users might copy the punctuation and asked that it be removed. Although punctuation can be used with imperative statements, it's a stylistic choice. If the Astronomer standard is to add punctuation to lists, I'll happily oblige.

>     - ENV AIRFLOW__SMTP__SMTP_STARTTLS: Enter True

Reviewer: ```suggestion
    - `ENV AIRFLOW__SMTP__SMTP_STARTTLS`: Enter `True`
```
Because `True` is a value that the user has to type out, we should use code formatting on it.
